endorsement/dao/shared_drive.py

```diff
@@ -487,6 +491,11 @@ def handle_missing_drives(self, missing_drive_ids, default_quota):
                 f"{missing_drive_count} > {self.missing_drive_threshold}"
             )
             return
+        elif missing_drive_count > self.missing_drive_notification:
+            notify_admin_missing_drive_count_exceeded(
+                missing_drive_count=missing_drive_count,
+                missing_drive_notification=self.missing_drive_notification,
```

Comment:
I don't actually see `notify_admin_missing_drive_count_exceeded` used in the template - what is this used for?

Should the `notify_admin_missing_drive_count_exceeded` call immediately above also have this `missing_drive_notification_` kwarg?

Reply:
those kwargs are effectively just template context, but for consistency and clarity that arg should definitely be include!  thanks!